src/main/webapp/app/quiz/participate/quiz.component.html

```diff
@@ -28,7 +41,7 @@ <h2 [ngSwitch]="mode">
                     {{totalScore}}
                 </div>
             </div>
-            <div *ngIf="!showingResult" id="remaining-time">
+            <div *ngIf="!showingResult" id="remaining-time" style="text-align: center; position: absolute; left: 42%">
```

Comment:
I do not like this inline style.
The `#remaining-time` element is already [styled here](https://github.com/ls1intum/ArTEMiS/blob/553c533a1dcb90c27a45a8028ccb4f5413fa02dd/src/main/webapp/content/scss/quiz.scss#L819).

Reply:
I just added them to see with the overlapping - if we decide to make the line break I would export the styles into the scss files